example/.gitignore

```diff
@@ -1,71 +0,0 @@
-# Miscellaneous
```

Comment:
Whats with all the deleted files? Please fix before someone can do a proper review